File: StringVeDiziOdevleri/Q02/main.c
#include <stdio.h>
#include <string.h>

int main()
{
    char dizgi[50];
    char altDizgi[50];

    printf("Bir dizgi girin ->");
    scanf("%s", dizgi);

    printf("Iceriginde kontrol edecek alt metni girin ->");
    scanf("%s", altDizgi);

    char *k = strstr(dizgi, altDizgi);

    if (k != NULL)
        printf("%s, %s icinde bulunmaktadir.", altDizgi, dizgi);
    else
        printf("Metin, alt metni icermemektedir.");

    return 0;
}

File: StringVeDiziOdevleri/Q02/main.py
dizgi = input("Bir dizgi girin ->")
altDizgi = input("Bir alt dizgi girin ->")

print(
    "dizgi, alt dizgiyi içerir."
    if dizgi.find(altDizgi) != -1
    else "dizgi, alt dizgiyi içermez"
)
